docs/sources/whatsnew/whats-new-in-v9-4.md

```diff
@@ -29,3 +29,9 @@ Description. Include an overview of the feature and problem it solves, and where
 We've improved the design and functionality of email templates to make template creation much easier and more customizable. The email template framework utilizes MJML to define and compile the final email HTML output. Sprig functions in the email templates provide more customizable template functions.
 
 {{< figure src="/static/img/docs/alerting/alert-templates-whats-new-v9.3.png" max-width="750px" caption="Email template redesign" >}}
+
+## Service account expiration dates
+
+We have included a new feature flag which allows us to set a new expiration date limit for all newly created service account tokens.
```

Comment:
Is this really behind a feature toggle? I though is was just a config option for service accounts

Reply:
Well, if you don't set anything (the default), this feature will be disabled.

Would you like to suggest an edit?

Reply:
I just think the wording is a bit confusing. This is not a feature flag but a config option under the server account section and with default value will not be used

Reply:
I reworded this considering your comments. PTAL